config/sites/admissions.uiowa.edu/views.view.majors.yml

```diff
@@ -386,6 +387,213 @@ display:
         - user.permissions
       tags:
         - 'config:field.storage.node.field_major_area_of_study'
+  available_majors:
+    id: available_majors
+    display_title: 'Available Majors'
+    display_plugin: page
+    position: 3
+    display_options:
+      title: 'Available Majors for 2 Plus 2 Students'
+      fields:
+        title:
+          id: title
+          table: node_field_data
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: node
+          entity_field: title
+          plugin_id: field
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: '<a href="/2plus2/majors/{{ title|lower|replace({''/'':''''})|replace({'' '':''-''}) }}">{{ title }}</a>'
```

Comment:
Good point...I think Views are an odd case any way you go about it, since they already mix queries/data with presentation.

We do already have a views preprocess going for the majors view--that might be a good place to handle it in this case, especially since it only needs to construct and add the link, and not actually rewrite the title. I'll take a look!

Reply:
@cory-skeers If you move this to a preprocess function, I would suggest also that it might be a good idea to create a constant for the string `/2plus2/majors/` and reference that so that if it changes we can just change the constant. Also, if it needs to be referenced anywhere else, we are not hardcoding it.